test/integration/user_visits_place_test.rb

```diff
@@ -40,4 +40,21 @@ class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
       assert_equal 1, Place.count
     end
   end
+
+  test "can comment" do
+    VCR.use_cassette("new place") do
+      login_create_place
+
+      page.fill_in 'input-comment', 
+        :with => 'The food is bad'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.